charms/slurmd/src/charm.py

```diff
@@ -326,6 +327,7 @@ def get_node(self) -> Dict[Any, Any]:
         node = {
             "node_parameters": {
                 **machine.get_slurmd_info(),
+                **CHARM_MAINTAINED_NODE_PARAMETERS,
```

Comment:
Since `CHARMED_MAINTAINED_NODE_PARAMETERS` is only referenced once here, and only `MemSpecLimit` is set in the constant, perhaps it would be better to set `MemSpecLimit` directly here for now:

```suggestion
                "MemSpecLimit": "1024",
```